feature/session/src/main/res/values/dimens.xml

```diff
@@ -3,7 +3,8 @@
     <dimen name="session_bottom_sheet_left_time_space">76dp</dimen>
     <dimen name="session_bottom_sheet_left_time_text_size">20dp</dimen>
     <dimen name="session_bottom_sheet_left_time_text_left">16dp</dimen>
-    <dimen name="session_bottom_sheet_left_time_text_top_min">22dp</dimen>
```

Comment:
I've removed `session_bottom_sheet_left_time_text_top_min` because it's not clear for what we should use it... If we should keep it for future use, please suggest it.